third_party/libusb/naclport/src/libusb_over_chrome_usb.h

```diff
@@ -139,6 +139,7 @@ class LibusbOverChromeUsb final : public LibusbInterface {
   int LibusbHandleEventsWithTimeout(
       libusb_context* context, int timeout_seconds);
 
+  std::unordered_map<int64_t, uint8_t> busNumbers_;
```

Comment:
nit: "bus_numbers_" (see https://google.github.io/styleguide/cppguide.html#Variable_Names)

Please also add a comment explaining what's the key and the value in this map.

Reply:
Done